plugins/rapid7_surface_command/plugin.spec.yaml

```
@@ -0,0 +1,73 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: rapid7_surface_command
+title: Rapid7 Surface Command
+description: Surface Command gives you full visibilty over your assets and identies across multiple technology platforms.
+version: 1.0.0
+supported_versions: ["v1.0.790"]
+vendor: rapid7
+status: []
+tags: [caasm]
+
+hub_tags:
+  use_cases: [data_enrichment]
+  keywords: [caasm, rapid7, surface_command, cloud_enabled]
+  features: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/example
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.rapid7.com
+
+cloud_ready: true
+enable_cache: false
+sdk:
+  type: slim
+  version: 5
```

We should update this to the newest version, at the moment we're on 6.3.3
```suggestion
  version: 6.3.3
```